extensions/sld/include/powsybl/iidm/extensions/sld/ConnectablePositionAdder.hpp

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_EXTENSIONS_SLD_CONNECTABLEPOSITIONADDER_HPP
+#define POWSYBL_IIDM_EXTENSIONS_SLD_CONNECTABLEPOSITIONADDER_HPP
+
+#include <powsybl/iidm/ExtensionAdder.hpp>
+#include <powsybl/iidm/extensions/sld/ConnectablePosition.hpp>
+#include <powsybl/stdcxx/optional.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace extensions {
+
+namespace sld {
+
+class ConnectablePositionAdder : public ExtensionAdder {
+public:
+    class FeederAdder {
+    public:
+        FeederAdder(ConnectablePositionAdder& parent, ConnectablePosition::OptionalFeeder& feeder);
+
+        ConnectablePositionAdder& add();
+
+        FeederAdder& withDirection(const ConnectablePosition::Direction& direction);
+
+        FeederAdder& withName(const std::string& name);
+
+        FeederAdder& withOrder(unsigned long order);
+
+    private:
+        std::string m_name;
+
+        stdcxx::optional<unsigned long> m_order;
+
+        stdcxx::optional<ConnectablePosition::Direction> m_direction;
+
+        ConnectablePositionAdder& m_parent;
+
+        ConnectablePosition::OptionalFeeder& m_feeder;
+    };
+
+public:
+    /**
+     * Constructor
+     */
+    explicit ConnectablePositionAdder(Extendable& extendable);
+
+    /**
+     * Destructor
+     */
+    ~ConnectablePositionAdder() noexcept override = default;
+
+    /**
+     * Copy constructor
+     */
+    ConnectablePositionAdder(const ConnectablePositionAdder&) = default;
```

Comment:
Add a variable name

Reply:
When defaulted/deleted, we do not write any parameter usually.

Reply:
Create an issue to remove the variable name for defaulted/deleted constructors and assignments.

Add a statement in the `wiki/coding-style` to explain that we do not give the variable name for defaulted/deleted constructors or assignment operators